test/python/common.py

```diff
@@ -368,5 +394,77 @@ def _is_ci_fork_pull_request():
     return False
 
 
-SKIP_ONLINE_TESTS = os.getenv('SKIP_ONLINE_TESTS', _is_ci_fork_pull_request())
-SKIP_SLOW_TESTS = os.getenv('SKIP_SLOW_TESTS', True) not in ['false', 'False', '-1']
+def get_test_options(option_var='QISKIT_TESTS'):
```

Comment:
This function is in charge of parsing and resolving implications between the test options. It could be better implemented in a different module or in its own class. While isolating, let's try to fix the following issues.

Reply:
b29c093